src/main/java/org/akaza/openclinica/control/managestudy/EventCrfLayerBuilder.java

```diff
@@ -130,7 +130,9 @@ void buildEvent() {
         html.append(subjectText).append(": ").append(studySubjectLabel).br();
         html.append(crfText).append(": ").append(crf.getName()).br();
         String statusText = eventCrfWorkflowStatus.getDisplayValue();
-        if (eventCrfBean.isRemoved()) {
+        // AC2: If a Form has status Removed = Yes or the event the form is in has status Removed = Yes,
+        // then "removed" will be displayed instead of the form workflow status for that form.
+        if (eventCrfBean.isRemoved() || (getStudyEvent() != null && getStudyEvent().isRemoved())) {
```

Comment:
These checks will throw NPE if the removed flag is null. Can you instead use something like BooleanUtils.isTrue(getStudyEvent().isRemoved())?

Reply:
nvm, the isRemoved method already does that